### PR TITLE
Create install and uninstall scripts

### PR DESCRIPTION
**Pull request by jbro**
I've created some install and uninstall scripts, and updated the readme-file to reflect the change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,4 +222,4 @@ Results:
 
 
 ## License
-`plsql_lexer` is licensed under the LGPL.
\ No newline at end of file
+`plsql_lexer` is licensed under the LGPL.
```
Comment (jbro) [inline README.md:225]: I don't know what happened here, I didn't touch that line :)

**Comment by jonheller1** [comment]
Awesome, thanks for your contributions!  I'll go over the code more thoroughly soon, I can see a few minor style issues I'd like to change for consistency.

For the "token_end_posistion" change, I had been thinking about adding something like "line_number" and "column_number" instead.  That seems to be a more traditional attribute of tokens, based on programs like ANTLR.  That would help with things like compiler warnings, that identify the line and column of the problem.  (Although I don't plan to build a compiler with this, so I'm not sure what the real use cases are.)

Would LINE_NUMBER and COLUMN_NUMBER be good enough for your purpose, or would the end position still be useful?

**Comment by jbro** [comment]
Thanks, for including my patch, I'm sorry, that I forgot to use branches, so I could have opened two pull requests, and made your repos less messy. I'll try and remember that next time :-)

I think the line and column number is a great idea. It would be, as you say, really useful for pointing to the code in warning and error messages. However I'm using your lexer for injecting code into the database, sort of like aspect oriented programming, so the token offset (maybe that's a better name), is exactly what i need :-)

**Comment by jonheller1** [comment]
I added LINE_NUMBER, COLUMN_NUMBER, and FIRST_CHAR_POSITION (in case something needs to be injected before the token).  These seem to be typical token attributes in other parsers and lexers.  I changed the type attribute name to LAST_CHAR_POSITION (the same as the package variable you added).  Hopefully I won't change the public procedures and attributes again, but I'm still not at a 1.0.0 so there's no guarantee yet.

Thanks again for your contribution!

**Pull request merged**